convert_hf_to_gguf.py

```diff
@@ -243,7 +293,7 @@ def extra_f16_tensors(self, name: str, new_name: str, bid: int | None, n_dims: i
 
         return False
 
-    def write_tensors(self):
+    def prepare_tensors_for_writing(self):
```

Comment:
Not sure about that name change, feels a bit verbose. Why not simply `prepare_tensors`?

Especially since it can be used for other purposes than "for writing", like preparing the tensors to then count and sum all their sizes.